ui/tutorial-renderer/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+
+android {
+    defaultConfig {
+        vectorDrawables.useSupportLibrary = true
+    }
+    dataBinding {
+        enabled = true
+    }
+}
+
+dependencies {
+    implementation project(':library:base')
+
+    implementation project(':ui:base-tool')
```

Comment:
`base-tool` is probably the wrong module to depend on here. `base-tool` has the framework for rendering tools being returned from the API.